README.md

```diff
@@ -93,6 +93,31 @@ scrypto test
 
 **Note:** The commands use the default account as transaction sender.
 
+## Playing with the simulator
+We provide a docker image so that you can build your blueprints 
```

Comment:
`Follow this guide to build reproducible WASM and RDP files for your Scrypto blueprints.`